src/Mod/Fem/femguiobjects/__init__.py

```diff
@@ -20,10 +20,10 @@
 # *                                                                         *
 # ***************************************************************************
 
-__title__ = "Fem Gui modules"
+__title__ = "FreeCAD FEM Gui modules"
```

Comment:
or just an empty file because a Python package directory needs an __init.py__ in femsolver they are all empty. See for example https://github.com/berndhahnebach/FreeCAD_bhb/blob/master/src/Mod/Fem/femsolver/__init__.py

Reply:
@berndhahnebach 
Ok i'll remove the __title__ and doxygen blocks...but should the file be totally empty or at least have a license block? like in `src/Mod/Fem/femguiobjects/__init__.py`

Reply:
@berndhahnebach please respond

Reply:
https://forum.freecadweb.org/viewtopic.php?f=10&t=31775